Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -440,7 +440,19 @@
             <trans-unit id="form.label_provider_reference">
                 <source>form.label_provider_reference</source>
                 <target>Provider Referenz</target>
-            </trans-unit>            
+            </trans-unit>
+            <trans-unit id="sonata.media.block.media">
+                <source>sonata.media.block.media</source>
+                <target>Medium</target>
```

Comment:
"Medium" is right? sounds weird?

"Media" is better IMO, what do you think @core23